interface/patient_file/encounter/diagnosis.php

```diff
@@ -33,29 +33,29 @@
 );
 
 if ($payment_method == "insurance") {
-	$payment_method = "insurance: " . $insurance_company;
+    $payment_method = "insurance: " . $insurance_company;
 }
 if (isset($mode)) {
-	if ($mode == "add") {
+    if ($mode == "add") {
 
-		// Get the provider ID from the new encounter form if possible, otherwise
```

Comment:
Note to self and others:  form_encounter has long needed an input for provider(s) when creating the encounter, vs waiting and using the billing provider assigned in the fee sheet!